webui/react/src/shared/components/Image/Image.test.tsx

```diff
@@ -0,0 +1,125 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { DarkLight } from 'shared/themes';
+
+import { ImageAlert, ImageEmpty, ImageWarning } from './Image';
+import type { Props } from './Image';
+
+const setupImageAlert = (props?: Props) => {
+  const view = render(<ImageAlert {...props} />);
+  return { view };
```

Comment:
**nit:** totally up to you, but I don't foresee any future properties getting added and returned besides `view`. It could probably just return it straight up instead of wrapped in an object.

Reply:
I agree that any properties wouldn't be added in the future, but if it happens, that would be painful to change everything. I just wanted to make it flexible, so I'll leave it